### PR TITLE
Setup react

### PR DESCRIPTION
**Pull request by MeqdamAlqudah**
1- Creat react app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+### Setup
+
+-click the code button in the repo and then download the files as a zip file
+
+#### To get a local copy up and running follow these simple example steps.
+
+- Click the green `Code` button on the repo and copy link
+- In your local PC, open your terminal in the folder you would like to clone the project.
+- Clone the repo with the command: `git clone (copied link)`; like so: `git clone https://github.com/MeqdamAlqudah/Leaderboard`
+- On the terminal, navigate into the directory like so: `cd Leaderboard`
```
Comment (TedLivist) [inline README.md:29]: - [ ] Kindly fix your typo error here. Your documentation refers to **`Leaderboard`** whereas you are working on a different project.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @MeqdamAlqudah,

Good job so far!
✔️ Apps works correctly
✔️ No console log errors
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TedLivist) [inline README.md]: - [ ] Kindly fix your typo error here. Your documentation refers to **`Leaderboard`** whereas you are working on a different project.

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @MeqdamAlqudah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**